### PR TITLE
[pre-commit] Fix style issues in test scripts under `tests/acl` folder

### PR DESCRIPTION
**Pull request by xwjiang-ms**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
pre-commit is a static analysis tool introduced recently. This tool is not able to do diff-only check. It checks the whole files touched by PR. We can't blame PR author for legacy issues. That's why currently the pre-commit check is only optional.

To ensure that we can make pre-commit a mandatory check for PRs submitted to this repository, we need to fix all the legacy issues complained by pre-commit.
#### How did you do it?
This change fixes the style issues of test scripts under the `tests/acl` folder

### CODE REVIEW

File: tests/acl/test_acl.py [hunk 1]
```diff
@@ -18,10 +17,10 @@
 from tests.common.helpers.assertions import pytest_require
 from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
 from tests.common.config_reload import config_reload
-from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py, run_garp_service, change_mac_addresses
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py, run_garp_service, change_mac_addresses   # noqa F401
 from tests.common.utilities import wait_until
-from tests.common.dualtor.dual_tor_mock import mock_server_base_ip_addr
-from tests.common.helpers.assertions import pytest_assert
+from tests.common.dualtor.dual_tor_mock import mock_server_base_ip_addr # noqa F401
+from tests.common.helpers.assertions import pytest_assert   # noqa F401
```
Comment (wangxin) [inline tests/acl/test_acl.py:23]: Probably this line can be removed.

**Comment by wangxin** [comment]
under `acl` folder -> under `tests/acl` folder?

**Comment by xwjiang-ms** [comment]
> under `acl` folder -> under `tests/acl` folder?

Yes, that would be more correctly

File: tests/acl/test_acl.py (hunk 1 above)
Comment (xwjiang-ms) [reply tests/acl/test_acl.py]: Yes, I think so

**Review by wangxin — APPROVED** [review]

**Pull request merged**